api/api.go

```diff
@@ -215,9 +217,12 @@ func (api *DatasetAPI) enablePublicEndpoints() {
 	api.get("/datasets/{dataset_id}/editions/{edition}/versions", api.getVersions)
 	api.get("/datasets/{dataset_id}/editions/{edition}/versions/{version}", api.getVersion)
 	api.get("/datasets/{dataset_id}/editions/{edition}/versions/{version}/metadata", api.getMetadata)
-	api.get("/datasets/{dataset_id}/editions/{edition}/versions/{version}/observations", api.getObservations)
 	api.get("/datasets/{dataset_id}/editions/{edition}/versions/{version}/dimensions", api.getDimensions)
 	api.get("/datasets/{dataset_id}/editions/{edition}/versions/{version}/dimensions/{dimension}/options", api.getDimensionOptions)
+
+	if api.enableObservationEndpoint {
```

Comment:
Consider adding a log here to say its enabled - could be useful debugging issues.

Reply:
Good point - I have added logs as suggested, as well as when the graph DB healthcheck is enabled.